internal/controller/nats/controller.go

```diff
@@ -200,7 +200,7 @@ func (r *Reconciler) initNATSInstance(ctx context.Context, nats *nmapiv1alpha1.N
 	log.Infof("Istio enabled on cluster: %t", istioExists)
 
 	// Check if NATS account secret exists.
-	accountSecretName := fmt.Sprintf("%s-secret", nats.Name)
+	accountSecretName := nats.Name + "-secret"
```

Comment:
Is this way of appending preferred by lint?

Reply:
it is... the linter is called `perfsprint`. It is faster and imho easier to read. But we can disable the linter and revert if you dont agree with the readability aspects. The performance is not critical for our usecase.